acquisitions/src/main/resources/thunderjet/consortia/reusable/updateItemOwnership.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Update Item Ownership
+
+  Background:
+    * url baseUrl
+
+  Scenario: Update Item Ownership
```

Comment:
In our reusables at the Scenario level we usually go for a different naming pattern, try something like `updateItemOwnership`, I remember Damien telling me this too back in June